src/components/Paginator/Paginator.tsx

```diff
@@ -33,6 +35,9 @@ export const Paginator = ({
         getPagesCount({itemsPerPage, totalItems, maxPages}),
     );
 
+    const {locale} = useContext(LocaleContext);
+    const blogPath = getBlogPath(locale?.pathPrefix || '');
```

Comment:
Now the logic of creating an url is divided into two components. Maybe we should put this code in the Pagination Item ?

Reply:
ok, I will do it